package.json

```diff
@@ -12,18 +13,30 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.1",
+    "@types/jest": "^26.0.20",
```

Comment:
Please run the following command:
`npm i -D @types/jest @types/node @types/react @types/react-dom`

In general, check the first answer on this stackoverflow question about [the difference between dependencies and devDependencies](https://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies)